### PR TITLE
CMakeLists.txt: Slight refactor

### PR DESCRIPTION
**Pull request by erysdren**
Added an install target and changed around the TARADINO_SUFFIX stuff to allow that to work. Up next I'm planning to include some icons and the `.desktop` file in the install target as well.

Edit: I'm also thinking of ditching the dedicated FindSDL2.cmake files and using the `FindPkgConfig` module instead.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 
 option(TARADINO_SHAREWARE "Build shareware version" OFF)
 option(TARADINO_WERROR "Enable -Werror on build" OFF)
+option(TARADINO_INSTALL "Enable install target" OFF)
```
Comment (fabiangreffrath) [inline CMakeLists.txt:13]: Thanks for this. However, I don't understand why we should need to make the install target optional?

File: misc/rott.desktop.in [hunk 2]
```diff
@@ -2,8 +2,8 @@
 Type=Application
 Name=Rise of the Triad
 Comment=A high quality, fast scrolling first-person perspective 3D action game
-Icon=rott
-TryExec=rott
-Exec=rott
+Icon=@TARADINO_EXEC@
```
Comment (fabiangreffrath) [inline misc/rott.desktop.in:5]: Could you add the following changes, please:
- Rename the file to taradino.desktop
- `Name=Taradino`
- `Comment=SDL2 port of Rise of the Triad`
- `Keywords=first;person;shooter;fps;3d;rott;rise;of;the;triad;`

Comment (fabiangreffrath) [reply misc/rott.desktop.in:5]: Could you also rename all the icons in the `misc/` directory, adjust the `misc/icons.mk` Makefile and remove the useless `runrott.sh` script?

**Comment by erysdren** [comment]
And what do you think about using the `FindPkgConfig` module instead of those FindSDL2.cmake files?

**Comment by fabiangreffrath** [comment]
It should work on our main target platforms, right? Not sure about MSVC, though.

**Comment by erysdren** [comment]
I just adjusted the manpage as well, and added it to the install target. Let me know if you have any notes.

File: doc/taradino.6.in [hunk 3]
```diff
@@ -76,9 +77,11 @@ Maximum time to count down from (next parameter is time in seconds).
 ?
 
 .SH AUTHORS
-ROTT was created and published as shareware by Apogee Software, Ltd. and was published commercially by FormGen, Inc.
+Rise of the Triad was created and published as shareware by Apogee Software, Ltd. and was published commercially by FormGen, Inc.
 .PP
-The primary contributors to the icculus.org port are: Steven Fuller, Ryan C. Gordon, John Hall and Dan Olson.
+The primary contributors to the Taradino port are: Steven Fuller, Ryan C. Gordon, John Hall, Dan Olson, Fabian Greffrath, and erysdren.
```
Comment (fabiangreffrath) [inline doc/taradino.6.in:82]: The primary contributors to the original icculus.org port are: ...
The authors of the Taradino port are: ...

Something like this. :wink:

**Review by fabiangreffrath — APPROVED** [review]
Thank you!

**Pull request merged**